nlptest/nlptest.py

```diff
@@ -20,6 +22,13 @@ class Harness:
     used to test the model. A report is generated with test results.
     """
     SUPPORTED_HUBS = ["spacy", "huggingface", "johnsnowlabs"]
+    DEFAULTS_DATASET = {
+        ("ner", "dslim/bert-base-NER", "huggingface"): "conll/sample.conll",
```

Comment:
@JulesBelveze since "ner" is the default task would you be able to make it so that users don't necessarily need to pass the task type when calling these specific NER models?

Reply:
@luca-martial hmmm you mean you want the parameter `task` to be defaulted to `"ner"` or just to infer that if one uses `"dslim/bert-base-NER", "huggingface"` then `task="ner"`?

Reply:
Looks like `task="ner"` as default wasn't implemented yet - @chakravarthik27 can you please revisit that, maybe we forgot to merge that change